### ISSUE TITLE
Accessibiity

### ISSUE THREAD

**Issue by cmart89** [cmart89/git#1]
#Errors(11)
- [x] Missing alternative textMore information

- [x] Image button missing alternative textMore information

- [x] Missing form labelMore information

- [ ] Document language missingMore information

- [ ]Empty buttonMore information

- [x] Empty linkMore information

**Comment by cmart89** [comment]
![image](https://cloud.githubusercontent.com/assets/10935229/19258570/af7def8e-8f2d-11e6-94e2-253d6a65da6a.png)
